test/suites/standard/validate-certificate-rotation.robot

```diff
@@ -0,0 +1,73 @@
+*** Settings ***
+Documentation       Tests related to upgrading MicroShift
+
+Resource            ../../resources/common.resource
+Resource            ../../resources/microshift-process.resource
+Library             Collections
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+
+*** Variables ***
+${USHIFT_HOST}                      ${EMPTY}
+${USHIFT_USER}                      ${EMPTY}
+${KUBE_SCHEDULER_CLIENT_CERT}       /var/lib/microshift/certs/kube-control-plane-signer/kube-scheduler/client.crt
+${OSSL_CMD}                         openssl x509 -noout -dates -in
+
+
+*** Test Cases ***
+Certificate Rotation
+    [Documentation]    Performs Certificate Expiration Rotation test
+    Check Cert Expiration Date    ${KUBE_SCHEDULER_CLIENT_CERT}    365
+    Go To Future Date    150
```

Comment:
Does this cause the certs to be regenerated? The way the test is written doesn't make me think it does, but maybe I'm missing something?

Reply:
OK, when I read the test, I interpreted it as "certs should live 365 days; move to the future; certs should live 365 days". It would be clearer if the argument to "Check Cert Expiration Date" was a date, instead of a duration.